files/common/scripts/gobuild.sh

```
@@ -70,6 +70,14 @@ while read -r line; do
     LD_EXTRAFLAGS="${LD_EXTRAFLAGS} -X ${line}"
 done < "${BUILDINFO}"
 
+# verify go version before build
+# NB. this was copied verbatim from Kubernetes hack
+minimum_go_version=go1.13 # supported patterns: go1.x, go1.x.x (x should be a number)
+IFS=" " read -ra go_version <<< "$(${GOBINARY} version)"
+if [[ "${minimum_go_version}" != $(echo -e "${minimum_go_version}\n${go_version[2]}" | sort -s -t. -k 1,1 -k 2,2n -k 3,3n | head -n1) && "${go_version[2]}" != "devel" ]]; then
+    echo "Warning: Detected that you are using a low version of GO. Istio requires ${minimum_go_version} or greater."
```

Is outputting a warning sufficient/useful? Seems like we should just stop the build instead.

---

the buiild is already going to fail if they don't use 1.13. I don't think there is any reason to fail early, since this check may have false positives